CI/appveyor.install.ps1

```diff
@@ -9,10 +9,10 @@ $global:ErrorActionPreference = "Stop"
 
 SetQtBaseDir "$logFile"
 
-$Env:PATH = "$CMakePath;C:\MinGW\bin;C:\MinGW\msys\1.0\bin;C:\Program Files\7-Zip;$Env:PATH"
+$Env:PATH = "$CMakePath;C:\MinGW\bin;C:\MinGW\msys\1.0\bin;C:\Program Files\7-Zip;C:\Program Files\ccache;$Env:PATH"
```

Comment:
Doesn't the `ccache.exe` need to be found in the path BEFORE any compiler that it is substituting for might be found?

Reply:
No, this is okay because the binary is called directly by the build system